Digitall.Warehouse.Application/Features/Categories/Commands/CreateCategoryCommandHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Digitall.Warehouse.Application.Abstractions.Messaging;
+using Digitall.Warehouse.Application.Abstractions.Persistence;
+using Digitall.Warehouse.Domain.Entities.Products;
+using Digitall.Warehouse.Domain.Shared;
+
+namespace Digitall.Warehouse.Application.Features.Categories.Commands;
+
+public class CreateCategoryCommandHandler : ICommandHandler<CreateCategoryCommand>
+{
+    private readonly IUnitOfWork _unitOfWork;
+
+    public CreateCategoryCommandHandler(IUnitOfWork unitOfWork)
+    {
+        _unitOfWork = unitOfWork;
+    }
+
+    public async Task<Result> Handle(
+        CreateCategoryCommand request,
```

Comment:
I prefer to use "Command" and "Query" for handlers instead of "request" but this is up to you.

Reply:
fair point but even better to use the name as the actual type, ex: `createCategoryCommand`
Will do the change in the feature branch if that's ok for you. For sure will be easier for me to rebase my code :)